app/controllers/items_controller.rb

```diff
@@ -1,3 +1,6 @@
 class ItemsController < ApplicationController
+  def index
+    @merchant = Merchant.find(params[:merchant_id])
+  end
```

Comment:
I'm curious about the use of `merchant_id` instead of `id` since it's a primary key. Did it still work properly using this?

Reply:
@awiley33 I also had to use merchant_id to get it to work.

Reply:
Okay interesting, do either of you know why? I don't see anywhere in the csv, model, or schema that label it as such

Reply:
I believe its because in our table we have it set to merchant_id rather then just id